### DISCUSSION TITLE
Suggested project ideas

### DISCUSSION THREAD

**Discussion by rishabhv12** [COST-SD/support#14] [Ideas]
Here are some project ideas with their tech stack.
Interesting one can join any projects. 

1. **Email Alerts on WhatsApp**   - > `Twilio Functions`,  `IMAP library` ,  `Javascript `

2. **Resume Builder Web Application**   - > `Node.js `, `Material UI`

3. **Online Code Editor**   ->   `Node.Js` , `React`

4. **TinyMCE  Text Editor**   - >  `Javascript` , `Plugins`

Repository  details will be updated soon .

**Comment by rishabhv12** [comment]
Comment down your name along with your interesting project idea .

**Reply by ankur0904** [reply]
NAME - Ankur Singh
Project Idea - A MERN web application which help us to manage our schedule and also we can set the remainder and web app remind us via email or message.

**Reply by rishabhv12** [reply]
Good idea Ankur .. you can work on this and if you need any help feel free to ask

**Reply by ankur0904** [reply]
Thank you sir for your support. 
I am working on it. I code some part of the web app. Please check it out.

https://www.linkedin.com/posts/ankur-singh-161458227_project-mernstack-reactjs-activity-7024435229821939714-jBIX?utm_source=share&utm_medium=member_desktop

**Reply by rishabhv12** [reply]
It's good , Keep it up
